app/views/camp/insert.php

```diff
@@ -0,0 +1,264 @@
+                                            </select><br>
+                                        </div>
+                                        <label for="ch_phone_no">Phone number </label>
+                                        <input id="ch_phone_no" class="form-control" placeholder="Enter Phone Number" name="ch_phone_no" required /><br>
+                                        <label for="ch_email">Email </label>
+                                        <input id="ch_email" placeholder="Enter Email" class="form-control" type="email" name="ch_email" required /><br>
+                                        <label for="ch_mailing_list">Mailing list </label>
+                                        <input id="ch_mailing_list" class="form-control" placeholder="Enter Mailing List" type="email" name="ch_mailing_list" /><br>
+                                        <label for="ch_mailing_address">Mailing address </label>
+                                        <input id="ch_mailing_address" class="form-control" placeholder="Enter Mailing Address" type="email" name="ch_mailing_address" /><br>
+                                        <label for="ch_gis_url">GIS URL </label>
+                                        <input id="ch_gis_url" class="form-control" placeholder="Enter GIS URL - iframe tag" name="ch_gis_url" /><br>    
+                                    </div>
+                                </div>
+                            <fieldset>
+                                <div class="panel panel-primary">
+                                    <div class="panel-heading" style="font-size: large; padding-bottom: 15px">
+                                        <b>Contact Details for Camp</b>
+                                        <div class="btn btn-info" id="show4" style="float: right">Hide</div>
+                                    </div>
+                                    <div class="panel-body" id="panelBody4">
+                                        <label for="phone_no">Phone number </label>
+                                        <input id="phone_no" class="form-control" placeholder="Enter Phone Number" name="phone_no" required /><br>
+                                        <label for="email">Email </label>
+                                        <input id="email" placeholder="Enter Email" class="form-control" type="email" name="email" required /><br>
+                                        <label for="mailing_list">Mailing list </label>
+                                        <input id="mailing_list" class="form-control" placeholder="Enter Mailing List" type="email" name="mailing_list" /><br>
+                                        <label for="mailing_address">Mailing address </label>
+                                        <input id="mailing_address" class="form-control" placeholder="Enter Mailing Address" type="email" name="mailing_address" /><br>
```

Comment:
This is not email address, this is physical address. 

Also it should be a textarea, not input.